obs_status.yml

```diff
@@ -0,0 +1,2147 @@
+  mag_aca_err: 0.1
+- agasc_id: 694561568
+  mag_aca: 9.8
+  mag_aca_err: 0.1
+- agasc_id: 720504096
+  mag_aca: 11.344
+  mag_aca_err: 0.1
+- agasc_id: 723651200
+  mag_aca: 12.0
+  mag_aca_err: 0.1
+- agasc_id: 723657296
+  mag_aca: 11.547
+  mag_aca_err: 0.1
+- agasc_id: 740693864
+  mag_aca: 11.875
+  mag_aca_err: 0.1
+- agasc_id: 763893760
+  mag_aca: 11.6
+  mag_aca_err: 0.1
+- agasc_id: 764150080
+  mag_aca: 12.0
+  mag_aca_err: 0.1
+- agasc_id: 771756656
+  mag_aca: 11.641
+  mag_aca_err: 0.1
+- agasc_id: 797847184
+  mag_aca: 12.3
+  mag_aca_err: 0.2
+- agasc_id: 798760200
+  mag_aca: 11.2
+  mag_aca_err: 0.1
+- agasc_id: 805966472
+  mag_aca: 11.703
+  mag_aca_err: 0.1
+- agasc_id: 821318496
+  mag_aca: 10.719
+  mag_aca_err: 0.1
+- agasc_id: 836372744
+  mag_aca: 11.609
+  mag_aca_err: 0.1
+- agasc_id: 836381648
+  mag_aca: 11.203
+  mag_aca_err: 0.1
+- agasc_id: 879768704
+  mag_aca: 10.859
+  mag_aca_err: 0.1
+- agasc_id: 891159184
+  mag_aca: 11.203
+  mag_aca_err: 0.1
+- agasc_id: 897878136
+  mag_aca: 10.75
+  mag_aca_err: 0.1
+- agasc_id: 898125104
+  mag_aca: 11.203
+  mag_aca_err: 0.1
+- agasc_id: 899950912
+  mag_aca: 11.688
+  mag_aca_err: 0.1
+- agasc_id: 901664112
+  mag_aca: 10.406
+  mag_aca_err: 0.1
+- agasc_id: 904010616
+  mag_aca: 10.902  # need to check, not from starcheck report
+  mag_aca_err: 0.1
+- agasc_id: 913441888
+  mag_aca: 10.031
+  mag_aca_err: 0.1
+- agasc_id: 915541320
+  mag_aca: 11.6
+  mag_aca_err: 0.1
+- agasc_id: 915939232
+  mag_aca: 12.906
+  mag_aca_err: 0.1
+- agasc_id: 962998672
+  mag_aca: 11.406
+  mag_aca_err: 0.1
+- agasc_id: 974791856
+  mag_aca: 11.375
+  mag_aca_err: 0.1
+- agasc_id: 1020796600
+  mag_aca: 11.5
+  mag_aca_err: 0.1
+- agasc_id: 1058938112
+  mag_aca: 10.906
+  mag_aca_err: 0.1
+- agasc_id: 1081092600
+  mag_aca: 11.0
+  mag_aca_err: 0.1
+- agasc_id: 1092619768
+  mag_aca: 11.288  # need to check, not from starcheck report
+  mag_aca_err: 0.1
+- agasc_id: 1105592952
+  mag_aca: 10.734
+  mag_aca_err: 0.1
+- agasc_id: 1108353064
+  mag_aca: 10.91  # need to check, not from starcheck report
+  mag_aca_err: 0.1
+- agasc_id: 1117787424
+  mag_aca: 11.172
+  mag_aca_err: 0.1
+- agasc_id: 1130649544
+  mag_aca: 8.062
+  mag_aca_err: 0.1
+- agasc_id: 1139171624
+  mag_aca: 10.297
+  mag_aca_err: 0.1
+- agasc_id: 1143744224
+  mag_aca: 10.6
+  mag_aca_err: 0.1
+- agasc_id: 1178221128
+  mag_aca: 10.031
+  mag_aca_err: 0.1
+- agasc_id: 1197635184
+  mag_aca: 10.578
+  mag_aca_err: 0.1
+obs:
+- agasc_id:
+  - 901664112
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2003:129:13:39:37.186
+  status: 1
+- agasc_id:
+  - 686557584
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2003:221:17:58:51.959
+  status: 1
+- agasc_id:
+  - 897878136
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2003:228:21:05:34.379
+  status: 1
+- agasc_id:
+  - 857375576
+  comments: Expected star position is >1000 arcsec away during the whole obs
```

Comment:
I don't immediately see what was going on in obsid 4469.  The post-facto guide props suggests that we were tracking star 857375848 in slot 6 instead of 857375576.  https://icxc.harvard.edu/aspect/mica_reports/04/04469/props.html Seems fine to just exclude the data for this star-obs combination.

Reply:
👍🏼 

Do you suggest modifying the comments?

Reply:
It is a good comment now but not sure if we also want shorthand to say that the star catalog data we have for it must be wrong.

Reply:
I updated the comments with that note about the catalog.  I think we can just safely exclude these star-obs observations without making code changes to special case them.  It is a separate question on if we should have some fixes in the starcheck database and timelines / kadi to get the catalogs right.